src/domain/geetest/error.ts

```diff
@@ -0,0 +1,6 @@
+export class GeetestError extends Error {
```

Comment:
`Geetest` is an implementation that domain should now know about. I think the component is a `CaptchaProvider`.